ui/src/App.js

```diff
@@ -1,8 +1,17 @@
 import Header from "./components/Header/Header";
+import { Routes, Route } from "react-router-dom";
+
 
 function App() {
   return (
-    <div>Hey</div>
+    <Routes>
```

Comment:
This should be moved to a separate folder name `Routes`. 

@pallabez  What would be the homepage? As we are not focusing on the Login screen as of now.

Reply:
Well, as we decided the login is not the priority right now then yes `dashboard`. We need to also highlight in UI till `login` is not their `dashboard` view is of the `super user` or `non-super user`.